packages/fxa-graphql-api/src/lib/server.ts

```diff
@@ -39,25 +58,42 @@ export async function createServer(
   const schema = await TypeGraphQL.buildSchema({
     container: Container,
     resolvers: [AccountResolver],
+    validate: false,
   });
+  const authHeader = config.authHeader.toLowerCase();
   const authUser = Container.get(SessionTokenAuth);
   const debugMode = config.env !== 'production';
-  const defaultContext = async ({ req }: any) => {
-    const bearerToken = req.headers[config.authHeader.toLowerCase()];
+  const defaultContext = async ({ req }: { req: Request }) => {
```

Comment:
Adding the correct type to the `req` object highlighted that a check needs to be done before assuming its a string.